src/main/scala/firrtl/ir/IR.scala

```diff
@@ -328,7 +333,7 @@ class IntWidth(val width: BigInt) extends Width with Product {
     case w: IntWidth => width == w.width
     case _ => false
   }
-  override def hashCode = width.toInt
+  override lazy val hashCode = width.toInt
```

Comment:
This one should be faster if not lazy, since computing this simple hash code is probably less costly than the lazy val bookkeeping.